core/blockchain.go

```diff
@@ -2168,7 +2168,7 @@ func (bc *BlockChain) ReadValidatorInformationAt(
 	if err != nil || state == nil {
 		return nil, errors.Wrapf(err, "at root: %s", root.Hex())
 	}
-	wrapper, err := state.ValidatorWrapper(addr)
+	wrapper, err := state.ValidatorWrapper(addr, false)
```

Comment:
Please hide this function and make two wrappers , one for true and false and put name of book intent right in function name

Reply:
done.